docs/WINE.md

```diff
@@ -0,0 +1,55 @@
+# The guide on using Wine with Box64
+
+<ins>W</ins>ine <ins>I</ins>s <ins>N</ins>ot an <ins>E</ins>mulator, but Box64 is. Box64 supports Wine in multiple ways. Before continuing reading, you should have a basic understanding of Wine.
+
+## Wine variants
+
+Wine ships in multiple forms:
+
+
+1. **x86 Wine**
+
+    The executable architecture of **x86 Wine** is x86, and it runs x32 Windows executables.
+
+    **x86 Wine** requires a 32-bit environment (read 32-bit Linux libraries), which is usually available on x86_64 machines because of the nature of the architecture.
+
+    However, when you try to run it via an emulator on RISC architectures, the situation is a bit different. For Arm64, there is armhf, which is a 32-bit environment, and there is also Box86, which is made for armhf, so you can use Box86 to run **x86 Wine**.
+
+    However, for RISC-V 64, LoongArch64, and many newer Arm64 cores, a 32-bit environment is not available. Box64 introduced a new mode called Box32, which can be enabled at compile time via a CMake option `BOX32`. It essentially emulates a 32-bit environment using the 64-bit environment, so it can be used to run **x86 Wine**. But be aware that Box32 is still highly experimental and unstable.
+
+2. **x86_64 Wine**
+
+    The executable architecture of **x86_64 Wine** is x86_64, and it runs x64 Windows executables.
```

Comment:
maybe you need to precise that you need a way to run 32bits stuffs there, we either box86 or box32?